### ISSUE TITLE
Property Buffer doesn't exist

### ISSUE THREAD

**Issue by evanlarkin10** [software-mansion/react-native-reanimated#2436]
### Expected behavior

Installation works and app can run without exceptions

### Actual behavior & steps to reproduce

Property Buffer doesn't exist exception prevents application from running

## Package versions

- React Native: 0.63.3
- React Native Reanimated: 2.2.0
- NodeJS:
- Xcode:12.5
- Java & Gradle: 1.8.0 & 6.5

## Affected platforms

- [x ] Android
- [x] iOS
- [] Web

**Comment by kacperkapusciak** [comment]
Hello @evanlarkin10 :wave:

From your description, I cannot infer what is the problem that you are facing. Could you please elaborate on the problem and provide a `snack`/`github repo`/`code snippet` that we could run and reproduce?

Cheers

**Comment by evanlarkin10** [comment]
Hey @kacperkapusciak,

To be honest it's been a while since I had the issue and I forget how I overcame it, I do remember it though. I had just installed the library as per the instructions and when I went to run the app afterwards I was greeted with the red screen error stating "Property _Buffer does not exist." I am not able to reproduce the issue, a lot of my application has changed since. I did try deleting node_modules, package-lock.json etc. So I'm not sure where the issue came from. Could have been something specific to my project or environment.
